clinicadl/data/dataloader/batchloader.py

```diff
@@ -39,7 +39,9 @@ def get_images(self) -> torch.Tensor:
             torch.Tensor: A tensor containing all the images from the batch.
         """
         # Return the images of the batch
-        return torch.cat([sample.sample for sample in self], dim=0).unsqueeze(1)
+        return torch.cat(
+            [sample.get_tensors()["image"] for sample in self], dim=0
+        ).unsqueeze(1)
```

Comment:
```suggestion
        torch.stack([sample.image.tensor for sample in self], dim=0)
```

``get_tensors`` will convert all the images to tensors. Here we only need one.

Besides, ``torch.stack`` is more suited than ``torch.cat``.